utility_functions/date_period.py

```diff
@@ -1,11 +1,23 @@
 from datetime import datetime, timedelta
+from typing import List, Tuple
 from connect2Databricks.read2Databricks import redshift_ccg_read, redshift_cdw_read
 from utility_functions.benchmark import timer
 from utility_functions.custom_errors import *
 from pytz import timezone
 import pytz
 
 
+def find_time_points(
+        start_date: str,
+        period_list: List[int],
+) -> Tuple[str]:
```

Comment:
Might be a good idea to add a quick docstring to make it clear that period_list should be a list of days to add to the start_date value to get the increased or decreased dates.

Reply:
Added docstring to the function just now!